lld/MachO/InputFiles.cpp

```diff
@@ -1879,6 +1879,9 @@ DylibFile::DylibFile(const InterfaceFile &interface, DylibFile *umbrella,
   installName = saver().save(interface.getInstallName());
   compatibilityVersion = interface.getCompatibilityVersion().rawValue();
   currentVersion = interface.getCurrentVersion().rawValue();
+  for (auto rpath : interface.rpaths())
```

Comment:
```suggestion
  for (const auto& rpath : interface.rpaths())
```

nit: This could  possibly trigger "performance-for-range clangtidy", so maybe should just avoid the unnecessary copy?